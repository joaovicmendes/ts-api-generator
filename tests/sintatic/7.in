// Deve reportar erro de falta de tipo
model [
    Student {
        name: string
        email: string
        ra: number
    }
    ExampleClass {
        student: Student
        grade:
    }
]
endpoints [
    Student {
        GET '/students'
        GET '/students/:ra'
        POST '/students'
        PUT '/students/:ra'
        DELETE '/students/:ra'
    }
    ExampleClass {
        GET 'class/:grade'
    }
]
