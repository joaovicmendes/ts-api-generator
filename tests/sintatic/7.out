Linha 11: erro sintático próximo a }
Compilação interrompida.
